websites/D/Doronime/metadata.json

```
@@ -1,16 +1,16 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.9",
 	"author": {
-		"name": "MaaKoo",
+		"name": "noxzym",
 		"id": "243728573624614912"
 	},
 	"service": "Doronime",
 	"description": {
 		"en": "Download anime, anime music, anime ongoing, or anime batch with Indonesian subtitle",
 		"id": "Unduh anime, anime musik, anime ongoing, atau anime batch dengan subtitle Indonesia"
 	},
-	"url": "doronime.id",
-	"version": "1.0.7",
+	"url": "doroni.me",
+	"version": "1.0.8",
```

This is more than a fix. So update it to:
```suggestion
	"version": "1.1.0",
```